oqs-template/generate.yml

```diff
@@ -1516,3 +1516,138 @@ sigs:
                     'pretty_name': 'ECDSA p521',
                     'oid': '1.3.9999.8.5.2',
                     'code_point': '0xfef5'}]
+  -
+    family: 'CROSS'
+    variants:
+      # RSDP 128
+      -
+        name: 'CROSSrsdp128balanced'
+        pretty_name: 'CROSS-rsdp-128-balanced'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_128_balanced'
+        oid: '1.3.6.1.4.1.62245.2.1.1'
+        code_point: '0xfef6'
+        enable: true
+      -
+        name: 'CROSSrsdp128fast'
+        pretty_name: 'CROSS-rsdp-128-fast'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_128_fast'
+        oid: '1.3.6.1.4.1.62245.2.1.2'
+        code_point: '0xfef7'
+        enable: false
+      -
+        name: 'CROSSrsdp128small'
+        pretty_name: 'CROSS-rsdp-128-small'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_128_small'
+        oid: '1.3.6.1.4.1.62245.2.1.3'
+        code_point: '0xfef8'
+        enable: false
+      # RSDP 192
+      -
+        name: 'CROSSrsdp192balanced'
+        pretty_name: 'CROSS-rsdp-192-balanced'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_192_balanced'
+        oid: '1.3.6.1.4.1.62245.2.1.4'
+        code_point: '0xfef9'
+        enable: false
+      -
+        name: 'CROSSrsdp192fast'
+        pretty_name: 'CROSS-rsdp-192-fast'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_192_fast'
+        oid: '1.3.6.1.4.1.62245.2.1.5'
+        code_point: '0xfefa'
+        enable: false
+      -
+        name: 'CROSSrsdp192small'
+        pretty_name: 'CROSS-rsdp-192-small'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_192_small'
+        oid: '1.3.6.1.4.1.62245.2.1.6'
+        code_point: '0xfefb'
+        enable: false
+      # RSDP 256
+      -
+        name: 'CROSSrsdp256balanced'
+        pretty_name: 'CROSS-rsdp-256-balanced'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_256_balanced'
+        oid: '1.3.6.1.4.1.62245.2.1.7'
+        code_point: '0xfefc'
+        enable: false
+      -
+        name: 'CROSSrsdp256fast'
```

Comment:
I am afraid you have to remove this algorithm here, @rtjk : As you assume, this has to do with its signature being longer than 64kB (I happen to know this limit as I instigated in OpenSSL the change from 16kB to the maximum permitted by spec at https://github.com/openssl/openssl/blob/e91384d5b0547bf797e2b44976f142d146c4e650/ssl/statem/statem_local.h#L34-L35 to allow more PQ algs to operate in OpenSSL). Thanks to @SWilson4 for triggering this CI!

Reply:
And in case you want to check the spec: It's section B.3.3 in https://www.rfc-editor.org/rfc/rfc8446 specifying maximum signature length in TLS.

Reply:
Got it, I just removed rsdp-256-fast and shifted the code-points accordingly.

Reply:
Thanks. Triggered a full re-test: https://github.com/open-quantum-safe/oqs-provider/actions/runs/10506136727

Reply:
But, @rtjk it seems you'd need to re-run [code formatting](https://github.com/open-quantum-safe/oqs-provider/blob/main/CONTRIBUTING.md#coding-style).

Reply:
Just done :)